coordinator/performance/prometheus/config.go

```diff
@@ -0,0 +1,14 @@
+package prometheus
```

Comment:
Missing license notice. I realize it is missing in a lot of places. We should do a full cleanup some time.

Reply:
@agnivade - can you open a ticket for this please? feel free to assign to me

Reply:
Done https://mattermost.atlassian.net/browse/MM-22000.